ivy/functional/frontends/numpy/mathematical_functions/rounding.py

```diff
@@ -32,6 +32,16 @@ def _ceil(
         ret = ivy.where(where, ret, ivy.default(out, ivy.zeros_like(ret)), out=out)
     return ret
 
+#round
+@to_ivy_arrays_and_back
+@handle_numpy_casting
+@from_zero_dim_arrays_to_scalar
+def round(
```

Comment:
This is being called but not being tested, you have to test this too adding it's test to it's respective file, same as you've done for the instance method. :slightly_smiling_face:

Reply:
I will do that thank you